src/pages/individual/Individual.jsx

```diff
@@ -122,6 +152,11 @@ export default function Individual() {
   const [merging, setMerging] = useState(false);
   const [deletingIndividual, setDeletingIndividual] = useState(false);
   const [deleteEncounterId, setDeleteEncounterId] = useState(null);
+  const adoptionNameIsLegitimate =
```

Comment:
Instead of doing this, why not just omit the fallback when deriving adoptionName. then below you can just do `{adoptionName && ...}`

Reply:
Great point.

Reply:
Done